.github/workflows/cypress-tests-runner.yml

```diff
@@ -260,3 +260,212 @@ jobs:
+
+      - name: Fetch keys
+        if: ${{ env.RUN_TESTS == 'true' }}
+        env:
+          TOML_PATH: "./config/development.toml"
+        run: |
+          LOCAL_ADMIN_API_KEY=$(yq '.secrets.admin_api_key' ${TOML_PATH})
+          echo "CYPRESS_ADMINAPIKEY=${LOCAL_ADMIN_API_KEY}" >> $GITHUB_ENV
+
+      - name: Install mold linker
+        if: ${{ runner.os == 'Linux' && env.RUN_TESTS == 'true' }}
+        uses: rui314/setup-mold@v1
+        with:
+          make-default: true
+
+      - name: Install Rust
+        if: ${{ env.RUN_TESTS == 'true' }}
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable 2 weeks ago
+          components: clippy
+
+      - name: Install sccache
+        if: ${{ env.RUN_TESTS == 'true' }}
+        uses: taiki-e/install-action@v2.41.10
+        with:
+          tool: sccache
+          checksum: true
+
+      - name: Install cargo-nextest
+        if: ${{ env.RUN_TESTS == 'true' }}
+        uses: taiki-e/install-action@v2.41.10
+        with:
+          tool: cargo-nextest
+          checksum: true
+
+      - name: Install Diesel CLI
+        if: ${{ env.RUN_TESTS == 'true' }}
+        uses: baptiste0928/cargo-install@v3.1.1
+        with:
+          crate: diesel_cli
+          features: postgres
+          args: --no-default-features
+
+      - name: Install Just
+        if: ${{ env.RUN_TESTS == 'true' }}
+        uses: taiki-e/install-action@v2.41.10
+        with:
+          tool: just
+          checksum: true
+
+      - name: Install LLVM-profdata
+        if: ${{ env.RUN_TESTS == 'true' }}
+        run: rustup component add llvm-tools-preview
+
+      - name: Install grcov
+        if: ${{ env.RUN_TESTS == 'true' }}
+        run: cargo install grcov
+
+      - name: Install Node.js
+        if: ${{ env.RUN_TESTS == 'true' }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+
+      - name: Install Cypress and dependencies
+        if: ${{ env.RUN_TESTS == 'true' }}
+        run: |
+          npm ci --prefix ./cypress-tests
+
+      - name: Run database migrations
+        if: ${{ env.RUN_TESTS == 'true' }}
+        shell: bash
+        env:
+          DATABASE_URL: postgres://db_user:db_pass@localhost:5432/hyperswitch_db
+        run: just migrate run --locked-schema
+
+      - name: Insert card info into the database
+        if: ${{ env.RUN_TESTS == 'true' }}
+        run: |
+          PGPASSWORD=db_pass psql --host=localhost --port=5432 --username=db_user --dbname=hyperswitch_db --command "\copy cards_info FROM '.github/data/cards_info.csv' DELIMITER ',' CSV HEADER;"
+
+      - name: Build project
+        if: ${{ env.RUN_TESTS == 'true' }}
+        env:
+          RUSTFLAGS: "-Cinstrument-coverage"
+        run: cargo build --package router --bin router --no-default-features --features=customer_v2,payment_methods_v2,v2 --jobs 3
```

Comment:
instead of this, can we use `just run_v2`?

Reply:
No, we can't run `just run_v2` because we need to build separately, and have the application run in the background when running tests.

We can consider adding a `build_v2` recipe if required.

Reply:
Yes let's add that